### PR TITLE
Update routing.md

### PR DESCRIPTION
**Pull request by leoj3n**
Move viewModel.logout from PageLogin to PageHome.

image

### CODE REVIEW

File: docs/can-guides/experiment/technology/routing.md [hunk 1]
```diff
@@ -506,15 +506,18 @@ Component.extend({
...
                 });
             }
 
             switch(this.routeData.page) {
                 case "home":
-                    return new PageHome({ });
+                    return new PageHome({
+						viewModel: {
+							logout: this.logout.bind(this)
+						}
```
Comment (chasenlehara) [inline docs/can-guides/experiment/technology/routing.md]: This needs less indentation.

File: docs/can-guides/experiment/technology/routing.md [hunk 2]
```diff
@@ -583,15 +586,18 @@ Component.extend({
...
```
Comment (chasenlehara) [inline docs/can-guides/experiment/technology/routing.md]: Same here.

File: docs/can-guides/experiment/technology/routing.md [hunk 3]
```diff
@@ -658,15 +664,18 @@ Component.extend({
...
```
Comment (chasenlehara) [inline docs/can-guides/experiment/technology/routing.md]: One more spot :)

**Comment by leoj3n** [comment]
@chasenlehara Fixed the indenting.

**Pull request merged**